### PR TITLE
`treehouses password change` (fixes #2072)

### LINKED ISSUE #2072: `treehouses password` needs `change` subsubcommand to change password
The current way to change the password is `treehouses password`, but this could lead to accidental password changes by misspelling other subsubcommands or using a non-existent subsubcommand.

This should be changed so that:
1. `treehouses password` will describe the `password` subcommand and its options.
2. `treehouses password change` will be the new way to change the password.

### PR DESCRIPTION
**Pull request by JLKwong**
I'm still debugging using a case inside a case.

### CODE REVIEW

File: modules/help.sh [hunk 1]
```diff
@@ -10,7 +10,8 @@ help [command]                            gives you a more detailed info about t
 verbose <on|off>                          makes each command print more output (might not work with treehouses remote)
 expandfs                                  expands the partition of the RPI image to the maximum of the SD card
 rename <hostname>                         changes hostname
-password <password|disable|enable>        changes the password for 'pi' user or disables/enables password authentication
+password <change> [password]              changes the password for 'pi' user or disables/enables password authentication
+         <disable|enable>
```
Comment (dogi) [inline modules/help.sh:14]: split comment same way as the options

File: modules/password.sh [hunk 2]
```diff
@@ -1,21 +1,31 @@
...
     disablepassword
     ;;
   "enable")
+    checkargn $# 1
     enablepassword
     ;;
   *)
-    chpasswd <<< "pi:$options"
-    echo "Success: the password has been changed"
+    log_and_exit1 "Error: Must use an option to 'change', 'disable', or 'enable'"
```
Comment (dogi) [inline modules/password.sh:28]: help_password

**Review by rjpadilla — APPROVED** [review]
Don't really understand how line 4 `checkargn $# 2` works, but it was added by @dogi, so I'm gonna go with it.

Other than that it all looks good.

File: modules/password.sh (hunk 2 above)
Comment (dogi) [reply modules/password.sh:29]: password_help ;)
sorry @JLKwong

**Review by dogi — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Pull request merged**